apps/root/src/pages/history/components/historyTable/index.tsx

```diff
@@ -249,16 +259,16 @@ const getTxEventRowData = (txEvent: TransactionEvent, intl: ReturnType<typeof us
     ) : txDate.startOf('day').equals(DateTime.now().minus({ days: 1 }).startOf('day')) ? (
       <FormattedMessage defaultMessage="Yesterday" description="yesterday" />
     ) : (
-      <>{txDate.toLocaleString(DateTime.DATE_SHORT)}</>
+      <>{txDate.toFormat('LL/dd/yyyy')}</>
```

Comment:
can we use toLocaleString for this? Or at least find some DateTime that fixes into this?

The main thing about this is that intl can parse base on localization and show those dates specific for each locale.

For example on use it does mm/dd/yyyy but on latam it does dd/mm/yyyy